mitosheet/mitosheet/step_performers/user_defined_import.py

```diff
@@ -80,6 +82,8 @@ def get_user_defined_importer_params_from_frontend_params(f: Callable, frontend_
                 user_defined_importer_params[param_name] = int(param_value)
             elif param_type == 'float':
                 user_defined_importer_params[param_name] = float(param_value)
+            elif param_type == 'bool':
+                user_defined_importer_params[param_name] = 'true' in param_value.lower()
```

Comment:
We could use the utility `is_env_variable_set_to_true` to do this check and support things like `T` also, which might be helpful